### PR TITLE
dev-python/*: add python 3.10 support

### PR DESCRIPTION
**Pull request by arthurzam**
Continuing scanning the CI output for `PythonCompatUpdate`, and calling `ebuild ... test`
Some of them even add support for `python3_9`

### CODE REVIEW

File: dev-python/python-editor/python-editor-1.0.4-r1.ebuild [hunk 1]
```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
-PYTHON_COMPAT=( pypy3 python3_{7..9} )
+PYTHON_COMPAT=( pypy3 python3_{7..10} )
```
Comment (mgorny) [inline dev-python/python-editor/python-editor-1.0.4-r1.ebuild:6]: How did you test it?

File: dev-python/pysvg/pysvg-0.2.2_p3.ebuild [hunk 2]
```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{7,8} )
+PYTHON_COMPAT=( python3_{7..10} )
```
[the remark this answers is not in the document]
Comment (arthurzam) [reply dev-python/pysvg/pysvg-0.2.2_p3.ebuild:6]: merged and tested the sample from upstream at location:
https://raw.githubusercontent.com/alorence/pysvg-py3/master/sample/tutorial.py

File: dev-python/python-editor/python-editor-1.0.4-r1.ebuild (hunk 1 above)
Comment (arthurzam) [reply dev-python/python-editor/python-editor-1.0.4-r1.ebuild:6]: Viewed the source code [here](https://github.com/fmoo/python-editor/blob/master/editor.py) (very simple therfore no problemt with 3.10)
and tried to run the test [here](https://github.com/fmoo/python-editor/blob/master/test.py).

**Comment by mgorny** [comment]
Ok, thanks.

**Pull request closed without merging**